.env

```diff
@@ -10,3 +10,4 @@ EXTEND_ESLINT=true
 
 REACT_APP_API_BASE_URL=https://ipa.dataocean.us
 REACT_APP_PEP_GRAPH_URL=https://dataocean.us/pep/
+REACT_APP_LANDING_URL=https://dataocean.us/
```

Comment:
? this variable already in .env
https://github.com/3v-workspace/DataOcean/blob/master/.env